ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/offerings/OfferingWithMultiPackagePaywall.kt

```diff
@@ -0,0 +1,50 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.testdata.offerings
+
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.PackageType
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.PaywallTemplate
+import com.revenuecat.purchases.ui.revenuecatui.data.testdata.TestData
+import java.net.URL
+
+internal val TestData.offeringWithMultiPackagePaywall: Offering
```

Comment:
I named this like this in iOS at the beginning when we just had 2 templates. I think `OfferingsWithTemplate2Paywall` might be more clear now?